src/index.css

```diff
@@ -134,6 +135,16 @@ code {
   border-radius: 6.5px !important;
 }
 
+.ant-radio-button-wrapper-checked {
+  -moz-transform: scale(1.05) !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5863170)